integration-tests/src/test/java/tachyon/client/FileOutStreamIntegrationTest.java

```diff
@@ -57,7 +58,9 @@ public static final void afterClass() throws Exception {
   @BeforeClass
   public static final void beforeClass() throws IOException {
     sLocalTachyonCluster = new LocalTachyonCluster(10000, 128, 128);
-    sLocalTachyonCluster.start();
+    TachyonConf tachyonConf = new TachyonConf();
+    tachyonConf.set(Constants.USER_FILE_BUFFER_BYTES, String.valueOf(BUFFER_BYTES));
```

Comment:
Okay, as long as the other tests do not rely on the size being the default value it should be fine.